desktop.css

```diff
@@ -0,0 +1,296 @@
+body {
```

Comment:
According to the project requirements, you need to use a media query in order to style your desktop version.

<img width="892" alt="Screen Shot 2022-06-03 at 15 40 36" src="https://user-images.githubusercontent.com/82426682/171949110-b5905f61-918e-451e-b277-4b75765ffe2e.png">

Kindly fix this.

Reply:
Hi! @vechicin could you be so kind to explain this to me? I don't quite get it. I added the media query in this way [Responsive web design](https://alistapart.com/article/responsive-web-design/) as explained in the [lesson](https://github.com/microverseinc/curriculum-html-css/blob/main/responsivness.md) this week. :thinking:

Reply:
Oh, I see, you added directly to your `<link>` tag. Adding a media query to a link counts as an inline style that goes against [HTML/CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). I apologize if the documentation is confusing on this subject, but please add your media query at the top of your CSS file.

Reply:
@vechicin I'll do that, thank you